frontend/src/features/admin/access-request/AccessRequestForm.tsx

```diff
@@ -56,28 +56,28 @@ export const AccessRequestForm: React.FunctionComponent<IAccessRequestFormProps>
           />
         </SectionField>
 
-        <SectionField label="First name" wideScreen>
+        <SectionField label="First name" labelWidth="2">
```

Comment:
changes not directly related to the documents screen. Changed for consistency and flexibility of the SectionField